src/server.js

```diff
@@ -0,0 +1,26 @@
+const restify = require('restify');
+const restifyPlugins = require('restify-plugins');
+
+function addNewDevice(name) {
```

Comment:
give a `// TODO:` above this function with a message about what you have to implement next. It's a nice way to indicate to fellow reviewers what you plan to do when you have a not quite ready feature.